shell/platform/windows/text_input_plugin_unittest.cc

```diff
@@ -574,13 +574,13 @@ TEST_F(TextInputPluginTest, TransformCursorRect) {
     arguments->AddMember("height", ime_height, allocator);
 
     auto message = codec.EncodeMethodCall(
-        {"TextInput.setMarkedTextRect", std::move(arguments)});
+        {"TextInput.setCaretRect", std::move(arguments)});
```

Comment:
Without having tested this locally, isn't this going to move the candidate window along under the cursor as you type? 

If so, that's pretty non-standard behaviour on Windows, at least for Japanese input -- not sure what the expected behaviour is for Sogou and others.